install/install.sh

```diff
@@ -114,7 +114,7 @@ fetch_and_unpack() {
   # I'd like to separate this into a fetch() and unpack() function, but I can't
   # figure out how to get bash functions to read STDIN/STDOUT from pipes
   if [ "${EXT}" = "tar.gz" ]; then
-    fetch "${URL}" | ${sudo} tar xzf "${VERBOSE}" - -C "${BIN_DIR}"
+    fetch "${URL}" | ${sudo} tar xz"${VERBOSE}"f - -C "${BIN_DIR}"
```

Comment:
Damn, I really messed it up, sorry 🤦 
Btw, maybe we can make it even more obvious:
```suggestion
    fetch "${URL}" | ${sudo} tar -xz"${VERBOSE}"f - -C "${BIN_DIR}"
```

Reply:
Nah, I was the reviewer there, the onus is on me. (Also, I'm a maintainer, making that doubly true).

So I don't know if this is still true, but back when I was first learning command line shenanigans a decade ago, I found that adding a `-` to tar options made them more order-sensitive, sometimes in ways that didn't agree with each other on different platforms.

Do we know that this problem no longer exists? Or perhaps it's an issue that I entirely imagined?

Reply:
As a simple example, `tar fxvz file.tgz` works on my system, while `tar -fxvz file.tgz` generates 
```
tar: You must specify one of the '-Acdtrux', '--delete' or '--test-label' options
Try 'tar --help' or 'tar --usage' for more information.
```

Reply:
I think `-f` needs to come last: `tar -xvzf` because that is the only one that receives argument. Does that work?

Reply:
Yes, I'm just not sure how that works on other platforms.

I guess another way to think of the question is this: can we think of a value of VERBOSE that will break the dashless form in a bad way (e.g. extracting the wrong file, passing inadvertent flags to `tar`) but won't break the dash form?

Reply:
> Yes, I'm just not sure how that works on other platforms.

Which platform you have? If this works on a Mac (I tested it, it does) it will work on other BSDs. The only other case would be GNU `tar` which I believe is fully backward compatible with BSD implementation.

> I guess another way to think of the question is this: can we think of a value of VERBOSE that will break the dashless form in a bad way (e.g. extracting the wrong file, passing inadvertent flags to tar) but won't break the dash form?

Well, that is a really hard question to answer... I guess we are a bit safer with dashed form because anything before `-f` would be treated as a flag, assuming that bad flag is better than operating on a bad file...